tests/strategies/test_parse.py

```diff
@@ -60,3 +68,55 @@ def test_cuds_parse(repo_dir: "Path") -> None:
     graph_comparison = graph_diff(graph_from_strategy, deser_graph)
     assert graph_comparison[1].serialize() == "\n"
     assert graph_comparison[2].serialize() == "\n"
+
+
+def test_cuds_parse_w_otelib(repo_dir: "Path") -> None:
+    """Test `application/cuds` parse strategy ."""
+    from oteapi.datacache import DataCache
+
+    # Create the otelib client with python as backend
+    from otelib import OTEClient
+    from rdflib import Graph
+    from rdflib.compare import graph_diff
+
+    from dlite_cuds.strategies.parse import CUDSParseStrategy
+
+    client = OTEClient("python")
+
+    # Specify and run the parse strategy as a pipeline in otelib
+    ontologypath = repo_dir / "tests" / "ontologies" / "chemistry.ttl"
+    cudspath = repo_dir / "tests" / "inputfiles_cuds2dlite" / "cuds.ttl"
+
+    config = {
+        "downloadUrl": cudspath.as_uri(),
+        "mediaType": "application/CUDS",
+        "configuration": {
+            "ontologyUrl": ontologypath.as_uri(),
+        },
+    }
+
+    source_and_parse = client.create_dataresource(**config)
+    pipeline = source_and_parse
+    data = pipeline.get()
+
+    # Get the data
+    parsed_data = eval(data.decode("utf-8"))
```

Comment:
Using the return value of the get() method is as far as I understand not how otelib was designed to be used. 

Before we start using the return value of the get() method this way in our tests, I think that we should have an internal discussion and agree on that this is an intended way to use otelib.

Reply:
I still don't understand why the parse strategy is returning a graph. Isn't the purpose of the parse strategy to parse a CUDS graph and populate the collection with instances obtained from the graph.

Reply:
Beacuse we are doing one thing at the time. Since we do not hate and oteapi_cuds like we have oteapi_dlite we decided to do it like this. It can be joined afterwards if we want, but I prefer to keep these two things apart for now during development.

Reply:
I have made the test skip in windows for now, and added an issue about it, depending on how the strategies end up.

Reply:
> Using the return value of the get() method is as far as I understand not how otelib was designed to be used.
> 
> Before we start using the return value of the get() method this way in our tests, I think that we should have an internal discussion and agree on that this is an intended way to use otelib.
> 
> I see that it is practical for testing, but accessing the internal state of a pipeline defeats the idea of pipelines being data documentation and may open for misuse of oteapi.
> 
> The alternative is to add a generate strategy to the pipeline.

I disagree. We have to test what the strategies do separately. If there is another way to do it, I am fine with it, but I do not think it is sufficient to just check that a strategy runs.